truelearn/learning/_engage_classifier.py

```diff
@@ -1,81 +1,28 @@
-from __future__ import annotations
+from typing import Dict
+from typing_extensions import Self, Any
```

Comment:
Im getting cant find reference Any in typing extensions ( I will need to debug this on my local machine perhaps)

Reply:
I should use `typing` instead of `typing_extensions` here.